mod-record-specifications-server/src/main/java/org/folio/rspec/service/SpecificationService.java

```diff
@@ -60,6 +61,20 @@ public SpecificationDtoCollection findSpecifications(Family family, FamilyProfil
     return specificationCollection.specifications(page.toList()).totalRecords(toIntExact(page.getTotalElements()));
   }
 
+  @Transactional
+  public SpecificationDto getSpecificationById(UUID specificationId, IncludeParam include) {
+    log.debug("findSpecificationById::id={}, include={}", specificationId, include);
+
+    return doForSpecificationOrFail(specificationId, specification -> switch (include) {
```

Comment:
Could you move this switch to separate method as it duplicates logic from findSpecifications